packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -51,6 +52,12 @@ class TestRenderingFlutterBinding extends BindingBase with Scheduler, Services,
     if (phase == EnginePhase.paint)
       return;
     renderView.compositeFrame();
+    if (phase == EnginePhase.composite)
```

Comment:
I reconciled by making this class define the same EnginePhase enumeration as widget_tester.dart. The beginFrame implementations should be consistent now. The widget_tester.dart implementation already knew about semantics, but it had a slightly different definition.